include/eve/concept/scalar.hpp

```diff
@@ -0,0 +1,52 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Contributors & Maintainers
+  SPDX-License-Identifier: MIT
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/kumi.hpp>
+#include <type_traits>
+#include <cstdint>
+#include <cstddef>
+
+namespace eve
+{
+  template<typename T>
+  concept plain_scalar  = std::is_same_v<T,float>         ||  std::is_same_v<T,double>
```

Comment:
a) we should have one_of concept
b) we should use std::same_as concept

Reply:
a) we have detaill:is_on_of<T>( types<U...>)  available